gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -514,6 +520,40 @@ public void createFile(Path filePath, byte[] data) throws IOException {
     }
   }
 
+  @Test
+  public void testCreateFSDataOutputStream() throws IOException {
+    Path path = new Path("/directory1/");
+
+    assertThrows("hadoopPath must not be null",IllegalArgumentException.class, () -> ghfs.create(null));
+    assertThrows("replication must be a positive integer: -1",IllegalArgumentException.class, () -> ghfs.create(path, (short) -1));
+    assertThrows("blockSize must be a positive integer: -1",IllegalArgumentException.class, () -> ghfs.create(path,true,-1,(short) 1,-1));
+  }
+
+  @Test
+  public void testRenameNullFile()  {
+    Path path = new Path("/directory1/");
+
+    assertThrows("src must not be null", NullPointerException.class, () -> ghfs.rename(null, null));
+    assertThrows("src must not be null", NullPointerException.class, () -> ghfs.rename(null, path));
+    assertThrows("dst must not be null", NullPointerException.class, () -> ghfs.rename(null, null));
```

Comment:
This is a duplicate test case, also I think there are no value to have `(null, null)` test case, if there are already `(path, null)` and `(null, path)` test cases.

Reply:
this scenario is only for having 100% branches coverage.